spec/shopify_transporter/shopify_transporter_spec.rb

```diff
@@ -306,6 +329,94 @@ def tmpfile(content)
     end
   end
 
+  describe ShopifyTransporter::Exporters::Exporter do
+    class SomePlatformExporter
+      def export
+        [{ foo: 'bar' }]
+      end
+    end
+
+    it 'writes exported data to file' do
+      in_temp_folder do
```

Comment:
Not sure why we need this.  Can we get rid of it?

Reply:
Oh yeah, that's from when we were generating output files and the test expectations were placed on the file contents. Now that we are writing to standard out, this is not required.